App/RequestList.js

```diff
@@ -98,16 +98,19 @@ class RequestList extends React.Component {
 
 
   getAvailableRequests(index){
-    var selectedIndex = index;
+
     return(
       <View style={styles.spacer}>
-      <TouchableHighlight onPress={this.props.goToVenueRequests.bind(this, selectedIndex)}>
+      <TouchableHighlight onPress={this.props.goToVenueRequests.bind(this)}>
```

Comment:
here is where goToVenueRequests is called...but binding anything else in the call yields nothing on homepage